framework/py/flwr/client/__init__.py

```diff
@@ -18,14 +18,12 @@
 from ..compat.client.app import start_client as start_client  # Deprecated
 from ..compat.client.app import start_numpy_client as start_numpy_client  # Deprecated
 from .client import Client as Client
-from .client_app import ClientApp as ClientApp
 from .numpy_client import NumPyClient as NumPyClient
 from .typing import ClientFn as ClientFn
 from .typing import ClientFnExt as ClientFnExt
 
 __all__ = [
     "Client",
-    "ClientApp",
```

Comment:
@panh99 @psfoley we can't remove this b/c it would be a breaking change for existing projects that do `from flwr.client import ClientApp`